zproject/urls.py

```diff
@@ -395,8 +395,10 @@
         {'POST': ('zerver.views.report.report_unnarrow_times', {'intentionally_undocumented'})}),
 
     # Used to generate a Zoom video call URL
-    url(r'^calls/create$', rest_dispatch,
-        {'GET': 'zerver.views.video_calls.get_zoom_url'}),
+    url(r'^calls/register_zoom_user$', rest_dispatch,
+        {'GET': 'zerver.views.video_calls.register_zoom_user'}),
+    url(r'^calls/complete_zoom_user$', rest_dispatch,
+        {'GET': 'zerver.views.video_calls.complete_zoom_user'}),
```

Comment:
My read of the Zoom API docs is that we need to also offer a "deregister_zoom_user` endpoint, that Zoom will post to when a user disables our integration.  Can you add that?  I think given how this integration works today, we can just return a 200 success without doing anything; if we later try to maintain the session for longer, we can adjust.